GDJS/Runtime/runtimescene.ts

```diff
@@ -14,13 +14,17 @@ namespace gdjs {
     _eventsFunction: null | ((runtimeScene: RuntimeScene) => void) = null;
     _instances: Hashtable<RuntimeObject[]>;
 
-    //Contains the instances living on the scene
+    // Contains the instances living on the scene
     _instancesCache: Hashtable<RuntimeObject[]>;
```

Comment:
The comments are not at the right place. I reorganized them in the custom PR. Avoid to change them to reduce conflicts.

Reply:
Alright, I'll remove them to avoid conflicts